homeassistant/components/nuki/__init__.py

```diff
@@ -88,6 +121,62 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         sw_version=info["versions"]["firmwareVersion"],
     )
 
+    async def handle_webhook(
+        hass: HomeAssistant, webhook_id: str, request: web.Request
+    ) -> web.Response:
+        """Handle webhook callback."""
+        try:
+            data = await request.json()
+        except ValueError:
+            return web.Response(status=HTTPStatus.BAD_REQUEST)
+
+        locks = hass.data[DOMAIN][entry.entry_id][DATA_LOCKS]
+        openers = hass.data[DOMAIN][entry.entry_id][DATA_OPENERS]
+
+        devices = [x for x in locks + openers if x.nuki_id == data["nukiId"]]
+        if len(devices) == 1:
+            devices[0].update_from_callback(data)
+
+        coordinator = hass.data[DOMAIN][entry.entry_id][DATA_COORDINATOR]
+        coordinator.async_set_updated_data(None)
+
+        return web.Response(status=HTTPStatus.OK)
+
+    webhook.async_register(
+        hass, DOMAIN, entry.title, entry.entry_id, handle_webhook, local_only=True
+    )
+
+    @callback
+    def _stop_nuki(_: Event):
+        """Stop and remove the Nuki webhook."""
+        webhook.async_unregister(hass, entry.entry_id)
+        try:
+            with async_timeout.timeout(10):
+                hass.async_add_executor_job(_remove_webhook, bridge, entry.entry_id)
```

Comment:
```suggestion
                await hass.async_add_executor_job(_remove_webhook, bridge, entry.entry_id)
```

Reply:
changed the method to `async` instead of `callback`.